src/bootstrap/bin/main.rs

```diff
@@ -7,10 +7,16 @@
 
 use std::env;
 
-use bootstrap::{Build, Config};
+use bootstrap::{Build, Config, Subcommand};
 
 fn main() {
     let args = env::args().skip(1).collect::<Vec<_>>();
     let config = Config::parse(&args);
+
+    if !config.config.exists() && !matches!(config.cmd, Subcommand::Setup { .. }) {
+        println!("warning: you have not made a `config.toml`");
+        println!("help: consider running `x.py setup` or copying `config.toml.example`");
+    }
```

Comment:
Similar to our other warning/lint-y type thing, I think duplicating this before/after probably makes sense.